### ISSUE TITLE
Combining specified layers of tiff

### ISSUE THREAD

**Issue by ITaluone** [dlemstra/Magick.NET#353]
### Prerequisites

- [x] I have written a descriptive issue title
- [x] I have verified that I am using the latest version of Magick.NET

### System Configuration
.net Framework 4.7.2

- Magick.NET version: 7.9.0.1
- Environment (Operating system, version and so on): Windows 10 1703 x64 

### Question
I want to do following:
```
MagickImageCollection tmpCollection = new MagickImageCollection();
string[] names = new string[]  { "layername1", "layername2", "layername3"};
foreach(string labelName in names) {
        tmpCollection.Add(anotherMagickImageCollection.Where(x=>x.Label == labelName).First());
 }
tmpCollection.TrimBounds();
return tmpCollection.ToBitmap();
```
But instead of returning an image with all added layers, it just returns tmpCollection[0].

What I have to do to return the expected?

Thank you

**Comment by dlemstra** [comment]
How do you want to combine the layers?

**Comment by ITaluone** [comment]
Hi :) 

Thanks for your quick reply.

I basically want to achieve that some "useless" layers (like background) are gone in the returning image/bitmap.

e.g. I have layers called:
Background --> white
Layer A --> Blue
Layer B --> Red
Layer C --> Blue "square"
Layer D --> "12"

I want get back one image, let's say B and C (at original coordinates).

![sample](https://user-images.githubusercontent.com/44049228/48901695-89206580-ee56-11e8-8ffc-357aa2c86996.jpg)
 
Do you understand what i mean?

Thank you

**Comment by ITaluone** [comment]
Ok after testing around, I think I've got it.

Tested is with these four methods:

- Combine() --> returns a completely "wrong" image 

- TrimBounds() --> also doesn't work for me

- Flatten() --> did not always work as expected

- Merge() --> this method worked in both cases

Now it would be interesting: what is the difference between these four methods? Every description mentions that this method is to combine photoshop layers to one image.

_Merge this collection into a single image. This is useful for combining Photoshop layers into a single image._
